contrib/packs/actions/pack_mgmt/download.py

```diff
@@ -16,7 +16,6 @@
 import os
```

Comment:
Please make sure there is good test coverage for all the changes. I don't see much tests for new code added but I haven't looked at it in detail. This is a dangerous file to make changes because of absence of tests. I'd say this is a PR blocker.